--- a/data/homebank.appdata.xml.in	2022-04-24 10:19:48.860593984 -0400
+++ b/data/homebank.appdata.xml.in	2022-04-24 10:20:12.461909047 -0400
@@ -2,6 +2,8 @@
 <application>
   <id type="desktop">homebank.desktop</id>
   <licence>GPL-2.0+</licence>
+  <name>HomeBank</name>
+  <summary>A personal accounting software</summary>
   <description>
     <_p>
       HomeBank is a free software (as in "free speech" and also as in "free beer") 
@@ -21,4 +23,8 @@
   </screenshots>
   <url type="homepage">http://homebank.free.fr</url>
   <updatecontact>homebank_at_free.fr</updatecontact>
+  <content_rating type="oars-1.1"/>
+  <releases>
+    <release date="2022-04-18" version="5.5.5"/>
+  </releases>
 </application>
