--- homebank-5.7/data/homebank.appdata.xml.in	2022-09-14 22:47:38.000000000 +0200
+++ homebank/data/homebank.appdata.xml.in	2023-09-07 08:21:56.464572832 +0200
@@ -1,29 +1,85 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<component type="desktop">
-  <id>fr.free.mdoyen.HomeBank.desktop</id>
+<component type="desktop-application">
+  <id>fr.free.Homebank</id>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0+</project_license>
   <name>HomeBank</name>
   <summary>Free, easy, personal accounting, for everyone</summary>
   <description>
-    <_p>
-      HomeBank is a free software (as in "free speech" and also as in "free beer") 
+    <p>
+      HomeBank is a free software (as in "free speech" and also as in "free beer")
       that will assist you to manage your personal accounting.
-    </_p>
-    <_p>
-      It is designed to easy to use and be able to analyse your personal finance 
+    </p>
+    <p>
+      It is designed to easy to use and be able to analyse your personal finance
       in detail using powerful filtering tools and beautiful graphs.
-    </_p>
-    <_p>
-      If you are looking for a completely free and easy way to manage your personal 
+    </p>
+    <p>
+      If you are looking for a completely free and easy way to manage your personal
       accounting then HomeBank should be the software of choice.
-    </_p>
+    </p>
   </description>
   <screenshots>
     <screenshot type="default">
-      <image width="624" type="default" height="351">http://homebank.free.fr/img/homebank-appdata.png</image>
+      <image>https://raw.githubusercontent.com/flathub/fr.free.Homebank/master/homebank.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://raw.githubusercontent.com/flathub/fr.free.Homebank/master/homebank-balance-report.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://raw.githubusercontent.com/flathub/fr.free.Homebank/master/homebank-vehicule-cost-report.png</image>
     </screenshot>
   </screenshots>
   <url type="homepage">http://homebank.free.fr</url>
+  <url type="bugtracker">https://launchpad.net/homebank/+filebug</url>
+  <url type="donation">http://homebank.free.fr/fr/donate.php</url>
   <update_contact>homebank@free.fr</update_contact>
+  <content_rating type="oars-1.1"/>
+  <releases>
+    <release date="2023-09-06" version="5.7">
+      <description>
+	<p>This is a new major release, with new features and bugfixes:</p>
+	<ul>
+	  <li>Cash flow forecast</li>
+	  <li>Budget report over time (month)</li>
+	  <li>Chart drill down for categories</li>
+	  <li>Home chart to show spending by month</li>
+	  <li>Home chart to show account balance total/time</li>
+	  <li>Budget report includes Category &amp; Subcategory</li>
+	  <li>Added balance mode for statistics total report</li>
+	  <li>Added fulfilled column for budget report</li>
+	  <li>And many more...</li>
+	</ul>
+        <p>See the ChangeLog for more details and read the documentation for additional info.</p>
+      </description>
+      <url>http://homebank.free.fr/ChangeLog</url>
+    </release>
+    <release date="2023-07-04" version="5.6.6">
+      <description>
+	<p>This is a bugfix release:</p>
+	<ul>
+	  <li>bugfix: #2027201 when ordering the transaction list by category, split ones are not ordered</li>
+	  <li>bugfix: #2026641 anonymise does not deal with account notes, start balance, overdraft and automatic assignment notes</li>
+	  <li>bugfix: #2026626 anonymise feature doesn't fully anonymise accounts name</li>
+	  <li>bugfix: #2026594 changing currency account with xfer faulty change target account currency</li>
+	  <li>bugfix: #2026184 statistics report doesn't always show total for parent category</li>
+	  <li>bugfix: #2024940 statistics faulty hide items with data when result is 0.0</li>
+	  <li>bugfix: #2024389 filter Status always show section should always be visible</li>
+	  <li>bugfix: #2019312 status column included in export detail txn from report break re-import</li>
+	</ul>
+      </description>
+    </release>
+    <release date="2023-06-21" version="5.6.5">
+      <description>
+	<p>This is a bugfix release:</p>
+	<ul>
+	  <li>bugfix: #2024243	when creating splits for income transactions, expenses are added as incomes regardless of amount sign</li>
+	  <li>bugfix: #2023388	account cannot be deleted message is not precise enough</li>
+	  <li>bugfix: #2022049	windows: displayed values overflow int32 in 5.6.4</li>
+	  <li>bugfix: #2019312	status column included in export detail txn from report break re-import</li>
+	  <li>bugfix: #2018039	lock icon for reconciled txn is not displayed in report detail</li>
+	</ul>
+      </description>
+    </release>
+  </releases>
 </component>
