--- homebank-5.7.4/data/homebank.appdata.xml.in	2022-09-14 22:47:38.000000000 +0200
+++ homebank/data/homebank.appdata.xml.in	2024-02-08 07:38:15.703979224 +0100
@@ -1,29 +1,185 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<component type="desktop">
-  <id>fr.free.mdoyen.HomeBank.desktop</id>
+<component type="desktop-application">
+  <id>fr.free.Homebank</id>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0+</project_license>
   <name>HomeBank</name>
-  <summary>Free, easy, personal accounting, for everyone</summary>
+  <summary>Personal accounting for everyone</summary>
+  <developer_name>Maxime Doyen</developer_name>
   <description>
-    <_p>
-      HomeBank is a free software (as in "free speech" and also as in "free beer") 
+    <p>
+      HomeBank is a free software (as in "free speech" and also as in "free beer")
       that will assist you to manage your personal accounting.
-    </_p>
-    <_p>
-      It is designed to easy to use and be able to analyse your personal finance 
+    </p>
+    <p>
+      It is designed to easy to use and be able to analyse your personal finance
       in detail using powerful filtering tools and beautiful graphs.
-    </_p>
-    <_p>
-      If you are looking for a completely free and easy way to manage your personal 
+    </p>
+    <p>
+      If you are looking for a completely free and easy way to manage your personal
       accounting then HomeBank should be the software of choice.
-    </_p>
+    </p>
   </description>
   <screenshots>
     <screenshot type="default">
-      <image width="624" type="default" height="351">http://homebank.free.fr/img/homebank-appdata.png</image>
+      <image>https://raw.githubusercontent.com/flathub/fr.free.Homebank/master/homebank.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://raw.githubusercontent.com/flathub/fr.free.Homebank/master/homebank-balance-report.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://raw.githubusercontent.com/flathub/fr.free.Homebank/master/homebank-vehicule-cost-report.png</image>
     </screenshot>
   </screenshots>
   <url type="homepage">http://homebank.free.fr</url>
+  <url type="bugtracker">https://launchpad.net/homebank/+filebug</url>
+  <url type="donation">http://homebank.free.fr/fr/donate.php</url>
   <update_contact>homebank@free.fr</update_contact>
+  <content_rating type="oars-1.1"/>
+  <releases>
+    <release date="2024-02-06" version="5.7.4">
+      <description>
+	<p>This is a bugfix release:</p>
+	<ul>
+	  <li>change: migrating to GTK4 preparation (GtkScrolledWindow)</li>
+	  <li>change: statistic report default type is now Total</li>
+	  <li>wish  : #2037200 add a used feature for tags like for payee/categories</li>
+	  <li>wish  : #2044850 revert 5.7.2 print single account (statement) fit to the page width</li>
+	  <li>wish  : #2051419 management dialogs to shows hidden items by default</li>
+	  <li>bugfix: template deletion was possible despite used into an account</li>
+	  <li>bugfix: dateentry was faultly trigger double parse/eval/update sequence</li>
+	  <li>bugfix: filter for report was faultly show always show items</li>
+	  <li>bugfix: update currency was not counting change to enable save</li>
+	  <li>bugfix: input a mindate > maxdate from the filter was possible with no warning</li>
+	  <li>bugfix: #2051440	export CSV not including memo field from split transactions</li>
+	  <li>bugfix: #2051349	add an existing category/payee/tag from manage dialog do not warn user</li>
+	  <li>bugfix: #2051307	export CSV not including remind status</li>
+	  <li>bugfix: #2050848	skip multiple Scheduled do not refresh as expected</li>
+	  <li>bugfix: #2048236	transfer are not included in forecast, balance mode</li>
+	  <li>bugfix: #2047647	cross-currency transfer do not propose target transaction</li>
+	  <li>bugfix: #2046157	info icon for transfers is not displayed</li>
+	  <li>bugfix: #2046032	date filter shows wrong date</li>
+	  <li>bugfix: #2043523	exclude zero-sum categories from Top Spending</li>
+	</ul>
+      </description>
+      <url>http://homebank.free.fr/ChangeLog</url>
+    </release>
+    <release date="2023-12-08" version="5.7.3">
+      <description>
+	<p>This is a bugfix release:</p>
+	<ul>
+	  <li>change: budget report added sign on categories like in manage dialog</li>
+	  <li>change: budget report only display fulfilled for budegt category</li>
+	  <li>bugfix: forecast was sometimes faultly done for excluded/closed account</li>
+	  <li>bugfix: date range 'last 12 months' start was not set to 1st day of -12 months</li>
+	  <li>bugfix: #2045299 'install for me only' end with error message when creating optional desktop shortcut</li>
+	  <li>bugfix: #2043886 portable version could prevent to store recent-file</li>
+	  <li>bugfix: #2043433 chart drill down Category link is not translated</li>
+	  <li>bugfix: #2043385 budget report forced displayed subcategories can have no parent</li>
+	  <li>bugfix: #2043366 budget forced category remains displayed with exp/inc filter</li>
+	  <li>bugfix: #2043223 budget report fulfilled column badly rounded display 99% vs 100%</li>
+	  <li>bugfix: #2042676 accound dialog exclude from any report do not refresh the graph after close</li>
+	  <li>bugfix: #2020181 mate: only french flag for payment icons</li>
+	  <li>wish  : #2042699 chart drill down shows the cat total when drawing subat</li>
+	  <li>wish  : #2042683 assigment rule dialog to be wider and/or size saved</li>
+	</ul>
+      </description>
+      <url>http://homebank.free.fr/ChangeLog</url>
+    </release>
+    <release date="2023-11-12" version="5.7.2">
+      <description>
+	<p>This is a bugfix release:</p>
+	<ul>
+	  <li>bugfix: crash was possible when no file loaded and change to main chart</li>
+	  <li>bugfix: #2043152	lock icon for transaction sometime disapear due to column resize</li>
+	  <li>bugfix: #2042771	currency don't update due to wrong api url since 5.7</li>
+	  <li>bugfix: #2042770	HomeBank closes after leaving "preferences" menu with no file open.</li>
+	  <li>bugfix: #2042668	remove/sanitize GTK listview quicksearch</li>
+	  <li>bugfix: #2042484	category popoverlist shows subcat income standalone its expense parent</li>
+	  <li>bugfix: #2042463	manage categories add subcategory fail after a search</li>
+	  <li>bugfix: #2042035	assign edit cancel faultly persist the rule search</li>
+	  <li>bugfix: #2040494	App crashes when exporting a report after I delete a category that contained data</li>
+	  <li>bugfix: #2040010	ISO 8601 date format not respected when exporting to clipboard or CSV</li>
+	  <li>bugfix: #2039995	budget by time do not filter txn type exp/inc</li>
+	  <li>bugfix: #2039493	statistics report crash when opened with a too wide date range</li>
+	  <li>wish  : #2042692	count and show number of selected transactions when suppressing them</li>
+	</ul>
+      </description>
+      <url>http://homebank.free.fr/ChangeLog</url>
+    </release>
+    <release date="2023-10-09" version="5.7.1">
+      <description>
+	<p>This is a bugfix release:</p>
+	<ul>
+	  <li>new: split dialog: add a confirm dialog before delete all split lines</li>
+	  <li>new: dateentry only rely on token and date order detection, glib failover</li>
+	  <li>change: split dialog: add info+tooltip icon next to the amount input</li>
+	  <li>change: split dialog: amount input is named according transaction type</li>
+	  <li>bugfix: dateentry d, dm, md was sometimes wrong</li>
+	  <li>bugfix: dateentry 2 year digits fix with 40/60 windowing</li>
+	  <li>bugfix: #2037597 top 10 Spending Category chart legend labels sometimes ellipsized</li>
+	  <li>bugfix: #2037468 "All Transactions" CSV export no longer includes account</li>
+	  <li>bugfix: #2037132 cannot create assignments from a transaction with empty memo</li>
+	  <li>bugfix: #2036703 budget report shows wrong Spent and Budget totals</li>
+	  <li>bugfix: #2036290 migrate libsoup-2.4 to libsoup-3.0</li>
+	  <li>bugfix: #2036270 no message when split limit is reached</li>
+	  <li>bugfix: #2036228 forecast only limited to time span set in preferences</li>
+	  <li>bugfix: #2036097 budget report date range from/to does not localized month name</li>
+	  <li>bugfix: #2035401 Main panel size not retained; account info concealed on start</li>
+	  <li>bugfix: #2035129 incorrect values in "tooltip" message for forcast view (subcategory)</li>
+	  <li>bugfix: #2034764 statistics Report Total Row not reflecting displayed amounts</li>
+	  <li>bugfix: #2034647 trend time Report Title does not match Interval</li>
+	  <li>bugfix: #2034618	statistics report sometimes crashes with half year interval </li>
+	  <li>wish: #2038601 "Schedule" and "Template" linked togglebuttons are not mutually exclusive (don't act like switchers)</li>
+	  <li>wish: #2034625	statistics Reportsorting by expense or income when using the Time Mode</li>
+	</ul>
+      </description>
+      <url>http://homebank.free.fr/ChangeLog</url>
+    </release>
+    <release date="2023-09-06" version="5.7">
+      <description>
+	<p>This is a new major release, with new features and bugfixes:</p>
+	<ul>
+	  <li>Cash flow forecast</li>
+	  <li>Budget report over time (month)</li>
+	  <li>Chart drill down for categories</li>
+	  <li>Home chart to show spending by month</li>
+	  <li>Home chart to show account balance total/time</li>
+	  <li>Budget report includes Category &amp; Subcategory</li>
+	  <li>Added balance mode for statistics total report</li>
+	  <li>Added fulfilled column for budget report</li>
+	  <li>And many more...</li>
+	</ul>
+        <p>See the ChangeLog for more details and read the documentation for additional info.</p>
+      </description>
+      <url>http://homebank.free.fr/ChangeLog</url>
+    </release>
+    <release date="2023-07-04" version="5.6.6">
+      <description>
+	<p>This is a bugfix release:</p>
+	<ul>
+	  <li>bugfix: #2027201 when ordering the transaction list by category, split ones are not ordered</li>
+	  <li>bugfix: #2026641 anonymise does not deal with account notes, start balance, overdraft and automatic assignment notes</li>
+	  <li>bugfix: #2026626 anonymise feature doesn't fully anonymise accounts name</li>
+	  <li>bugfix: #2026594 changing currency account with xfer faulty change target account currency</li>
+	  <li>bugfix: #2026184 statistics report doesn't always show total for parent category</li>
+	  <li>bugfix: #2024940 statistics faulty hide items with data when result is 0.0</li>
+	  <li>bugfix: #2024389 filter Status always show section should always be visible</li>
+	  <li>bugfix: #2019312 status column included in export detail txn from report break re-import</li>
+	</ul>
+      </description>
+    </release>
+    <release date="2023-06-21" version="5.6.5">
+      <description>
+	<p>This is a bugfix release:</p>
+	<ul>
+	  <li>bugfix: #2024243	when creating splits for income transactions, expenses are added as incomes regardless of amount sign</li>
+	  <li>bugfix: #2023388	account cannot be deleted message is not precise enough</li>
+	  <li>bugfix: #2022049	windows: displayed values overflow int32 in 5.6.4</li>
+	  <li>bugfix: #2019312	status column included in export detail txn from report break re-import</li>
+	  <li>bugfix: #2018039	lock icon for reconciled txn is not displayed in report detail</li>
+	</ul>
+      </description>
+    </release>
+  </releases>
 </component>
