--- homebank-5.9/data/homebank.appdata.xml.in	2022-09-14 22:47:38.000000000 +0200
+++ homebank/data/homebank.appdata.xml.in	2025-04-11 15:59:43.369945446 +0200
@@ -1,29 +1,403 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<component type="desktop">
-  <id>fr.free.mdoyen.HomeBank.desktop</id>
+<component type="desktop-application">
+  <id>fr.free.Homebank</id>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0+</project_license>
   <name>HomeBank</name>
-  <summary>Free, easy, personal accounting, for everyone</summary>
+  <summary>Personal accounting for everyone</summary>
+  <developer_name>Maxime Doyen</developer_name>
+  <launchable type="desktop-id">homebank.desktop</launchable>
   <description>
-    <_p>
-      HomeBank is a free software (as in "free speech" and also as in "free beer") 
+    <p>
+      HomeBank is a free software (as in "free speech" and also as in "free beer")
       that will assist you to manage your personal accounting.
-    </_p>
-    <_p>
-      It is designed to easy to use and be able to analyse your personal finance 
+    </p>
+    <p>
+      It is designed to easy to use and be able to analyse your personal finance
       in detail using powerful filtering tools and beautiful graphs.
-    </_p>
-    <_p>
-      If you are looking for a completely free and easy way to manage your personal 
+    </p>
+    <p>
+      If you are looking for a completely free and easy way to manage your personal
       accounting then HomeBank should be the software of choice.
-    </_p>
+    </p>
   </description>
   <screenshots>
     <screenshot type="default">
-      <image width="624" type="default" height="351">http://homebank.free.fr/img/homebank-appdata.png</image>
+      <image>https://raw.githubusercontent.com/flathub/fr.free.Homebank/master/homebank.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://raw.githubusercontent.com/flathub/fr.free.Homebank/master/homebank-balance-report.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://raw.githubusercontent.com/flathub/fr.free.Homebank/master/homebank-vehicule-cost-report.png</image>
     </screenshot>
   </screenshots>
   <url type="homepage">http://homebank.free.fr</url>
+  <url type="bugtracker">https://launchpad.net/homebank/+filebug</url>
+  <url type="donation">http://homebank.free.fr/fr/donate.php</url>
   <update_contact>homebank@free.fr</update_contact>
+  <content_rating type="oars-1.1"/>
+  <releases>
+    <release date="2025-04-10" version="5.9">
+      <description>
+	<p>This is a major release with new features, and bugfix...</p>
+	<ul>
+	  <li>new   : new svg default iconset</li>
+	  <li>new   : new symbolic svg iconset</li>
+	  <li>new   : budget tabview: added import/export menu</li>
+	  <li>new   : budget tabview: added clearall menu</li>
+	  <li>new   : reinforced error handling when loading a .xhb file</li>
+	  <li>new   : preference: amount color can be herited from chart palette</li>
+	  <li>new   : added a ledger tool to mark category sign with amount don't match</li>
+	  <li>change: xhb data file format (v1.6)</li>
+	  <li>change: internal data/display flag separation</li>
+	  <li>change: icon name rework</li>
+	  <li>change: simplified ledger icon status column (reconciled merged with lock)</li>
+	  <li>change: manage account icons was moved to a left column for consistency</li>
+	  <li>change: the preference pages have been organized on 2 levels</li>
+	  <li>change: the last used preferences page is remembered during the session</li>
+	  <li>change: relayout of edit assignment dialog</li>
+	  <li>wish  : #2101100 budget report option to exclude subcat from unbudgeted line</li>
+	  <li>wish  : #2089475 inherit while managing scheduled transactions</li>
+	  <li>wish  : #2083175 balance report to propose an option to include/exclude transfer</li>
+	  <li>wish  : #2079857 budget report option to restrict &lt;= today</li>
+	  <li>wish  : #2073032 statistic time report detail to focus on row or cell (specific month)</li>
+	  <li>wish  : #2070444 possibility to print/export scheduled transactions</li>
+	  <li>wish  : #2068845 duplicate a transaction split line</li>
+	  <li>wish  : #2068663 include split transaction memos in completion</li>
+	  <li>wish  : #2067399 Schedule Transaction for the nth weekday of a Month</li>
+	  <li>wish  : #2066993 scheduled payment on last day of month</li>
+	  <li>wish  : #2066110 refactor EUR feature for future euro adopters</li>
+	  <li>wish  : #2065633 option to keep internal transfer date sync</li>
+	  <li>wish  : #2065355 making "remind" status independent of "reconciled" status</li>
+	  <li>wish  : #2064958 "Period to date" date range</li>
+	  <li>wish  : #2061227 a feature to prevent add/import prior last reconciled date</li>
+	  <li>wish  : #2052314 option of background color for future transactions</li>
+	  <li>wish  : #2042038 assignment more visible search option (case and regex)</li>
+	  <li>wish  : #2042037 manage assignment list search option icons (case and regex)</li>
+	  <li>wish  : #1938412 inline calculation in txn/split amount field</li>
+	  <li>wish  : #1875100 option to import in pending status and approve later</li>
+	  <li>wish  : #1756407 remove -|+ of amount widget + empty if focus</li>
+	  <li>bugfix: post scheduled from main window do not enable save</li>
+	  <li>bugfix: #2106307 broken link in the Preferences Dialog </li>
+	  <li>bugfix: #2106027 delete then show usage make a crash in tags manage</li>
+	  <li>bugfix: #2105387 no changes in manage accounts faulty count changes</li>
+	  <li>bugfix: #2104162 Wrong balance values in Balance Report after changing the interval</li>
+	  <li>bugfix: #2102726 Unexpected dates from 'Next payout' automatic posting</li>
+	  <li>bugfix: #2101050 transaction sign change with a transfer between accounts</li>
+	  <li>bugfix: #2100875 payment filter revert to mobile phone after reopen saved file</li>
+	  <li>bugfix: #2100731 budget report time list print result label are incorrect</li>
+	  <li>bugfix: #2100489 windows: date problem if you change the short format in regional settings</li>
+	  <li>bugfix: #2099918 scheduled with week-end before/after gap post date is incorrect</li>
+	  <li>bugfix: #2099843 budget chart wrong color when spend = budget</li>
+	  <li>bugfix: #2098359 portable version truncate recently-used.xbel</li>
+	</ul>
+      </description>
+      <url>https://www.gethomebank.org/ChangeLog</url>
+    </release>
+    <release date="2024-12-28" version="5.8.6">
+      <description>
+	<p>This is a maintenance release.</p>
+	<ul>
+	  <li>bugfix: fix crash when print your accounts</li>
+	  <li>bugfix: ledger similar marked txn were persisted after close</li>
+	  <li>bugfix: ledger similair marked txn now display their group</li>
+	  <li>bugfix: repbalance first column title was not set to interval</li>
+	  <li>bugfix: #2091004 statistics time report totals line with exact 0 is not displayed</li>
+	  <li>bugfix: #2090668 close after start wizard not not let choose filename to save</li>
+	  <li>bugfix: #2090183 the export CSV functionality does not take into account the frac digits property of the currency</li>
+	  <li>bugfix: #2089407 trendtime/balance report clear filter don't clear properly</li>
+	  <li>bugfix: #2087750 save indicator (*) not showing after editing multiple transfert</li>
+	  <li>bugfix: #2085376 future/remind main window bottom list column not updated</li>
+	  <li>bugfix: #2083127 inherit operations should not preserve colour flags</li>
+	  <li>bugfix: #2079801 alert about budget usage when move/delete a category</li>
+	  <li>bugfix: #2060159 window width and height size is reset to a small size after demaximizing on startup</li>
+	  <li>wish  : #2086475 add menu keyboard shortcut to import files</li>
+	</ul>
+      </description>
+      <url>https://www.gethomebank.org/ChangeLog</url>
+    </release>
+    <release date="2024-10-05" version="5.8.5">
+      <description>
+	<p>This is a maintenance release.</p>
+	<ul>
+	  <li>bugfix: date button shortened in ledger book</li>
+	</ul>
+      </description>
+      <url>https://www.gethomebank.org/ChangeLog</url>
+    </release>
+    <release date="2024-10-05" version="5.8.4">
+      <description>
+	<p>This is a maintenance release.</p>
+	<ul>
+	  <li>bugfix: prevent some problems with xfer and both legder window opened</li>
+	  <li>bugfix: #2083124 account rename doesn't allow case correction</li>
+	  <li>bugfix: #2081574 windows: budget report date fields not showing in their entirety</li>
+	  <li>bugfix: #2081379 planned split operations are not modified after a category merge</li>
+	  <li>bugfix: #2080864 HomeBank CLI --version or -V no longer works in linux</li>
+	  <li>bugfix: #2080756 balance change from ledger toolbar not updated for xfer target</li>
+	  <li>bugfix: #2080032 your accounts print/export hidden columns</li>
+	  <li>bugfix: #2079884 ledger window snap half left oversize</li>
+	  <li>bugfix: #2036404 grouped budget categories not showing total amount</li>
+	</ul>
+      </description>
+      <url>https://www.gethomebank.org/ChangeLog</url>
+    </release>
+    <release date="2024-09-09" version="5.8.3">
+      <description>
+	<p>This is a maintenance release.</p>
+	<ul>
+	  <li>bugfix: #2066535 cannot see a 1 sub-category in Stats Rept donut view</li>
+	  <li>bugfix: #2076458 saved filter date ranges going wrong</li>
+	  <li>bugfix: #2076474 dark mode applied live, but not on app startup or when exiting preferences dialog</li>
+	  <li>bugfix: #2077180 budget report category do not sum their subcat budget in 5.8.2</li>
+	  <li>bugfix: #2078281 csv export crash</li>
+	  <li>bugfix: #2078679 category appeard Empty and crashes Homebank on Linux</li>
+	</ul>
+      </description>
+      <url>https://www.gethomebank.org/ChangeLog</url>
+    </release>
+    <release date="2024-08-10" version="5.8.2">
+      <description>
+	<p>This is a maintenance release.</p>
+	<ul>
+	  <li>bugfix: #2073805 switching saved filter with same date range doesn't refresh</li>
+	  <li>bugfix: #2073233 vehicle cost distance/volume is rounded to distance and always xx,0</li>
+	  <li>bugfix: #2071648 possibility for positive or negative budget values gone</li>
+	  <li>bugfix: #2069152 windows: CFA protected folder save fail with no dialog warning</li>
+	  <li>bugfix: #2068825 budget report type filter do not work anymore</li>
+	  <li>bugfix: #2068664 filter text option only work when both memo and number are filled</li>
+	  <li>bugfix: #2068634 copy/paste a txn do not enable save</li>
+	  <li>bugfix: #2067855 transfer from one account to another does not sync statuse even with "Sync transfer Status" checked</li>
+	  <li>bugfix: #2067440 payment settings not retained</li>
+	  <li>bugfix: #2066539 budget Report - unbudgeted includes transactions for subcats where main cat has a budget</li>
+	  <li>bugfix: #2062021 transaction date may change if system shortdate is not dd/MM/yy</li>
+	</ul>
+      </description>
+      <url>https://www.gethomebank.org/ChangeLog</url>
+    </release>
+    <release date="2024-05-26" version="5.8.1">
+      <description>
+	<p>This is a maintenance release with bugfix...</p>
+	<ul>
+	  <li>bugfix: #2066161 "Raw amount" check menu is not saved</li>
+	  <li>bugfix: #2065955 scheduled transactions not automatically posted before weekend</li>
+	  <li>bugfix: #2065929 saved filter does not show txn with remind status</li>
+	  <li>bugfix: #2065781 balance not updated after edit a transaction in ledger</li>
+	  <li>bugfix: #2065770 display bug of next date column in template dialog</li>
+	  <li>bugfix: #2065758 incorrect window title for credit card type</li>
+	  <li>bugfix: #2065740 scheduled transactions are broken for "Next Payout"</li>
+	  <li>bugfix: #2065628 windows: crash after open recent file no more existing</li>
+	  <li>bugfix: #2065625 save button stay disabled after edit a transaction</li>
+	  <li>bugfix: #2065592 changing status to none from txn dialog fail</li>
+	  <li>bugfix: #2065551 embedded help file misses some new files and images</li>
+	  <li>bugfix: #2065531 statistics report with tags filter show extra tags</li>
+	</ul>
+      </description>
+      <url>https://www.gethomebank.org/ChangeLog</url>
+    </release>
+    <release date="2024-05-14" version="5.8">
+      <description>
+	<p>This is a major release with new features, and bugfixes...</p>
+	<ul>
+	  <li>new   : statistics report: category sign is now also displayed</li>
+	  <li>new   : added a tooltip to display active filter in all report windows</li>
+	  <li>change: migrating to GTK4 preparation (GdkEvent struct)</li>
+	  <li>change: migrating to GTK4 preparation (GtkApplication, GtkApplicationWindow)</li>
+	  <li>change: migrating to GTK4 preparation (GtkMenu*, GtkToolbar) for ledger</li>
+	  <li>change: 'select action for target xfer' creation cancel now get back to txn dialog</li>
+	  <li>change: reports: moved the collapse/expand buttons to bottom of the list</li>
+	  <li>change: example file addition: split, tags, loan, flags, life-energy</li>
+	  <li>change: the Info field is named as it should have originally: Number</li>
+	  <li>change: added missing enums for most action of combobox</li>
+	  <li>wish  : #2064754 yours account should always display a tooltip for consistency</li>
+	  <li>wish  : #2064520 enable ledger txn reconciled > cleared, like for none</li>
+	  <li>wish  : #2063416 index for importing csv files</li>
+	  <li>wish  : #2060588 widths of memo -v- amount field in split transactions</li>
+	  <li>wish  : #2059733 preference to use the template list when edit a transaction</li>
+	  <li>wish  : #2058696 enrich account transactions list window title with account number and institution name</li>
+	  <li>wish  : #2058566 budget Report - detail pane size not adjustable</li>
+	  <li>wish  : #2056654 add thousand separator when print 'your accounts'</li>
+	  <li>wish  : #2055533 Payment column in auto-assignment window</li>
+	  <li>wish  : #2054906 "Cancel" button on prompt for a target txn when adding xfer should not create a transaction</li>
+	  <li>wish  : #2051758 add a help tip when input amount range in filter</li>
+	  <li>wish  : #2045514 Balance report to exclude transfers when multiple selected accounts</li>
+	  <li>wish  : #2044601 preference to always prompt for a target txn when adding xfer</li>
+	  <li>wish  : #2042674 idiot-proof preference for "Sync transfert Status"</li>
+	  <li>wish  : #2038753 copy transaction to clipboard to paste external apps</li>
+	  <li>wish  : #2037039 amend "Spending By Month" to have income option &amp; both income/expense by month</li>
+	  <li>wish  : #2023696 budget report to show not-budgeted amount</li>
+	  <li>wish  : #2017625 transfer txn widget From vs To field ordering to be static</li>
+	  <li>wish  : #2017437 reorderable columns in the Scheduled/Template window</li>
+	  <li>wish  : #2017436 scheduled/template window show internal transfers "To" account in Payee column</li>
+	  <li>wish  : #2016317 columns for status icon and payment icon in scheduled/template</li>
+	  <li>wish  : #2002177 automatic GTK dark theme switching for Linux (using the Freedesktop standard)</li>
+	  <li>wish  : #1910935 statistic report by accounts groups</li>
+	  <li>wish  : #1909255 report exclude income in transfers</li>
+	  <li>wish  : #1867979 internal transfer edit dialog to show/change date in both accounts</li>
+	  <li>wish  : #1867498 add "Life Energy" attribute</li>
+	  <li>wish  : #1817274 add more paymode + user order position + disable</li>
+	  <li>wish  : #1674016 assignment to assign tags</li>
+	  <li>wish  : #1600356 add colour flag/group to transaction</li>
+	  <li>wish  : #613894 favorite saved filter list to use in report and ledger show all</li>
+	  <li>bugfix: multiple txn edit tag was not displaying the tag list</li>
+	  <li>bugfix: time report quick filter item list were displaying too much columns</li>
+	  <li>bugfix: #2064839 post of today due date txn do not happen</li>
+	  <li>bugfix: #2063145 top spending pie chart subcategory is not sorted descending</li>
+	  <li>bugfix: #2063135 statistics details export menu items sensitivity update problem</li>
+	  <li>bugfix: #2061979 cross-currency transfer still do not propose target transaction</li>
+	  <li>bugfix: #2060159 window width and height size is reset to an invalid small size after demaximizing on startup</li>
+	  <li>bugfix: #2059709 filter exclusion of payment=none hide transfer</li>
+	  <li>bugfix: #2055101 windows: recent file Clear keep deleted/renamed files</li>
+	  <li>bugfix: #2052678 home time chart spending by month should show net expense</li>
+	  <li>bugfix: #2052304 budget should warn/fix when we input positive number for expense</li>
+	</ul>
+      </description>
+      <url>https://www.gethomebank.org/ChangeLog</url>
+    </release>
+    <release date="2024-02-06" version="5.7.4">
+      <description>
+	<p>This is a bugfix release:</p>
+	<ul>
+	  <li>change: migrating to GTK4 preparation (GtkScrolledWindow)</li>
+	  <li>change: statistic report default type is now Total</li>
+	  <li>wish  : #2037200 add a used feature for tags like for payee/categories</li>
+	  <li>wish  : #2044850 revert 5.7.2 print single account (statement) fit to the page width</li>
+	  <li>wish  : #2051419 management dialogs to shows hidden items by default</li>
+	  <li>bugfix: template deletion was possible despite used into an account</li>
+	  <li>bugfix: dateentry was faultly trigger double parse/eval/update sequence</li>
+	  <li>bugfix: filter for report was faultly show always show items</li>
+	  <li>bugfix: update currency was not counting change to enable save</li>
+	  <li>bugfix: input a mindate > maxdate from the filter was possible with no warning</li>
+	  <li>bugfix: #2051440	export CSV not including memo field from split transactions</li>
+	  <li>bugfix: #2051349	add an existing category/payee/tag from manage dialog do not warn user</li>
+	  <li>bugfix: #2051307	export CSV not including remind status</li>
+	  <li>bugfix: #2050848	skip multiple Scheduled do not refresh as expected</li>
+	  <li>bugfix: #2048236	transfer are not included in forecast, balance mode</li>
+	  <li>bugfix: #2047647	cross-currency transfer do not propose target transaction</li>
+	  <li>bugfix: #2046157	info icon for transfers is not displayed</li>
+	  <li>bugfix: #2046032	date filter shows wrong date</li>
+	  <li>bugfix: #2043523	exclude zero-sum categories from Top Spending</li>
+	</ul>
+      </description>
+      <url>http://homebank.free.fr/ChangeLog</url>
+    </release>
+    <release date="2023-12-08" version="5.7.3">
+      <description>
+	<p>This is a bugfix release:</p>
+	<ul>
+	  <li>change: budget report added sign on categories like in manage dialog</li>
+	  <li>change: budget report only display fulfilled for budegt category</li>
+	  <li>bugfix: forecast was sometimes faultly done for excluded/closed account</li>
+	  <li>bugfix: date range 'last 12 months' start was not set to 1st day of -12 months</li>
+	  <li>bugfix: #2045299 'install for me only' end with error message when creating optional desktop shortcut</li>
+	  <li>bugfix: #2043886 portable version could prevent to store recent-file</li>
+	  <li>bugfix: #2043433 chart drill down Category link is not translated</li>
+	  <li>bugfix: #2043385 budget report forced displayed subcategories can have no parent</li>
+	  <li>bugfix: #2043366 budget forced category remains displayed with exp/inc filter</li>
+	  <li>bugfix: #2043223 budget report fulfilled column badly rounded display 99% vs 100%</li>
+	  <li>bugfix: #2042676 accound dialog exclude from any report do not refresh the graph after close</li>
+	  <li>bugfix: #2020181 mate: only french flag for payment icons</li>
+	  <li>wish  : #2042699 chart drill down shows the cat total when drawing subat</li>
+	  <li>wish  : #2042683 assigment rule dialog to be wider and/or size saved</li>
+	</ul>
+      </description>
+      <url>http://homebank.free.fr/ChangeLog</url>
+    </release>
+    <release date="2023-11-12" version="5.7.2">
+      <description>
+	<p>This is a bugfix release:</p>
+	<ul>
+	  <li>bugfix: crash was possible when no file loaded and change to main chart</li>
+	  <li>bugfix: #2043152	lock icon for transaction sometime disapear due to column resize</li>
+	  <li>bugfix: #2042771	currency don't update due to wrong api url since 5.7</li>
+	  <li>bugfix: #2042770	HomeBank closes after leaving "preferences" menu with no file open.</li>
+	  <li>bugfix: #2042668	remove/sanitize GTK listview quicksearch</li>
+	  <li>bugfix: #2042484	category popoverlist shows subcat income standalone its expense parent</li>
+	  <li>bugfix: #2042463	manage categories add subcategory fail after a search</li>
+	  <li>bugfix: #2042035	assign edit cancel faultly persist the rule search</li>
+	  <li>bugfix: #2040494	App crashes when exporting a report after I delete a category that contained data</li>
+	  <li>bugfix: #2040010	ISO 8601 date format not respected when exporting to clipboard or CSV</li>
+	  <li>bugfix: #2039995	budget by time do not filter txn type exp/inc</li>
+	  <li>bugfix: #2039493	statistics report crash when opened with a too wide date range</li>
+	  <li>wish  : #2042692	count and show number of selected transactions when suppressing them</li>
+	</ul>
+      </description>
+      <url>http://homebank.free.fr/ChangeLog</url>
+    </release>
+    <release date="2023-10-09" version="5.7.1">
+      <description>
+	<p>This is a bugfix release:</p>
+	<ul>
+	  <li>new: split dialog: add a confirm dialog before delete all split lines</li>
+	  <li>new: dateentry only rely on token and date order detection, glib failover</li>
+	  <li>change: split dialog: add info+tooltip icon next to the amount input</li>
+	  <li>change: split dialog: amount input is named according transaction type</li>
+	  <li>bugfix: dateentry d, dm, md was sometimes wrong</li>
+	  <li>bugfix: dateentry 2 year digits fix with 40/60 windowing</li>
+	  <li>bugfix: #2037597 top 10 Spending Category chart legend labels sometimes ellipsized</li>
+	  <li>bugfix: #2037468 "All Transactions" CSV export no longer includes account</li>
+	  <li>bugfix: #2037132 cannot create assignments from a transaction with empty memo</li>
+	  <li>bugfix: #2036703 budget report shows wrong Spent and Budget totals</li>
+	  <li>bugfix: #2036290 migrate libsoup-2.4 to libsoup-3.0</li>
+	  <li>bugfix: #2036270 no message when split limit is reached</li>
+	  <li>bugfix: #2036228 forecast only limited to time span set in preferences</li>
+	  <li>bugfix: #2036097 budget report date range from/to does not localized month name</li>
+	  <li>bugfix: #2035401 Main panel size not retained; account info concealed on start</li>
+	  <li>bugfix: #2035129 incorrect values in "tooltip" message for forcast view (subcategory)</li>
+	  <li>bugfix: #2034764 statistics Report Total Row not reflecting displayed amounts</li>
+	  <li>bugfix: #2034647 trend time Report Title does not match Interval</li>
+	  <li>bugfix: #2034618	statistics report sometimes crashes with half year interval </li>
+	  <li>wish: #2038601 "Schedule" and "Template" linked togglebuttons are not mutually exclusive (don't act like switchers)</li>
+	  <li>wish: #2034625	statistics Reportsorting by expense or income when using the Time Mode</li>
+	</ul>
+      </description>
+      <url>http://homebank.free.fr/ChangeLog</url>
+    </release>
+    <release date="2023-09-06" version="5.7">
+      <description>
+	<p>This is a new major release, with new features and bugfixes:</p>
+	<ul>
+	  <li>Cash flow forecast</li>
+	  <li>Budget report over time (month)</li>
+	  <li>Chart drill down for categories</li>
+	  <li>Home chart to show spending by month</li>
+	  <li>Home chart to show account balance total/time</li>
+	  <li>Budget report includes Category &amp; Subcategory</li>
+	  <li>Added balance mode for statistics total report</li>
+	  <li>Added fulfilled column for budget report</li>
+	  <li>And many more...</li>
+	</ul>
+        <p>See the ChangeLog for more details and read the documentation for additional info.</p>
+      </description>
+      <url>http://homebank.free.fr/ChangeLog</url>
+    </release>
+    <release date="2023-07-04" version="5.6.6">
+      <description>
+	<p>This is a bugfix release:</p>
+	<ul>
+	  <li>bugfix: #2027201 when ordering the transaction list by category, split ones are not ordered</li>
+	  <li>bugfix: #2026641 anonymise does not deal with account notes, start balance, overdraft and automatic assignment notes</li>
+	  <li>bugfix: #2026626 anonymise feature doesn't fully anonymise accounts name</li>
+	  <li>bugfix: #2026594 changing currency account with xfer faulty change target account currency</li>
+	  <li>bugfix: #2026184 statistics report doesn't always show total for parent category</li>
+	  <li>bugfix: #2024940 statistics faulty hide items with data when result is 0.0</li>
+	  <li>bugfix: #2024389 filter Status always show section should always be visible</li>
+	  <li>bugfix: #2019312 status column included in export detail txn from report break re-import</li>
+	</ul>
+      </description>
+    </release>
+    <release date="2023-06-21" version="5.6.5">
+      <description>
+	<p>This is a bugfix release:</p>
+	<ul>
+	  <li>bugfix: #2024243	when creating splits for income transactions, expenses are added as incomes regardless of amount sign</li>
+	  <li>bugfix: #2023388	account cannot be deleted message is not precise enough</li>
+	  <li>bugfix: #2022049	windows: displayed values overflow int32 in 5.6.4</li>
+	  <li>bugfix: #2019312	status column included in export detail txn from report break re-import</li>
+	  <li>bugfix: #2018039	lock icon for reconciled txn is not displayed in report detail</li>
+	</ul>
+      </description>
+    </release>
+  </releases>
 </component>
